language-extensions/index.d.ts

```diff
@@ -1,2 +1,56 @@
 declare function $multi<T extends any[]>(...values: T): MultiReturn<T>;
 declare type MultiReturn<T extends any[]> = T & { readonly " __multiBrand": unique symbol };
```

Comment:
If you want to you can rename `MultiReturn` to `LuaMultiReturn` as discussed on discord. But we could also do that in a separate PR.

Reply:
I plan to do a PR to fix some issues with MultiReturn, so I figured I'd rename it in that.